pkg/controller/directvolumemigration/rsync_test.go

```diff
@@ -84,6 +84,14 @@ func getFakeCompatClient(obj ...k8sclient.Object) compat.Client {
 	return client
 }
 
+func getFakeCompatClientFor124(obj ...k8sclient.Object) compat.Client {
+	fakecompat.SetMajor(1)
```

Comment:
If I am not wrong, this is only changing the variable defined in the fakeClient package, its not really changing the version of the client.

Reply:
Thanks for catching this! You are right we are not building a fake client with a specific version.